libs/@hashintel/design-system/src/icon-arrow-right.tsx

```diff
@@ -1,7 +1,7 @@
 import { SvgIcon, SvgIconProps } from "@mui/material";
 import { FunctionComponent } from "react";
 
-export const ArrowRightIcon: FunctionComponent<SvgIconProps> = (props) => {
+export const IconArrowRight: FunctionComponent<SvgIconProps> = (props) => {
```

Comment:
Probably, although we're not going to have export subpaths in `design-system` so it will make no difference to consumers, will just be internal organization of the package – will leave it to some future dev to put them in an icon folder when there are too many to leave at the root